src/base/system_parameter.c

```diff
@@ -1395,19 +1395,19 @@ int PRM_FILTER_PRED_MAX_CACHE_CLONES = 10;
 static int prm_filter_pred_max_cache_clones_default = 10;
 static unsigned int prm_filter_pred_max_cache_clones_flag = 0;
 
-int PRM_LIST_QUERY_CACHE_MODE = 0;
-static int prm_list_query_cache_mode_default = 0;	/* disabled */
+int PRM_LIST_QUERY_CACHE_MODE = 1;
+static int prm_list_query_cache_mode_default = 1;
 static int prm_list_query_cache_mode_upper = 2;
 static int prm_list_query_cache_mode_lower = 0;
-static unsigned int prm_list_query_cache_mode_flag = 0;
+static unsigned int prm_list_query_cache_mode_flag = 1;
 
-int PRM_LIST_MAX_QUERY_CACHE_ENTRIES = -1;
-static int prm_list_max_query_cache_entries_default = -1;	/* disabled */
-static unsigned int prm_list_max_query_cache_entries_flag = 0;
+int PRM_LIST_MAX_QUERY_CACHE_ENTRIES = 200;
+static int prm_list_max_query_cache_entries_default = 200;
+static unsigned int prm_list_max_query_cache_entries_flag = 1;
 
-int PRM_LIST_MAX_QUERY_CACHE_PAGES = -1;
-static int prm_list_max_query_cache_pages_default = -1;	/* infinity */
-static unsigned int prm_list_max_query_cache_pages_flag = 0;
+int PRM_LIST_MAX_QUERY_CACHE_PAGES = 1000;
+static int prm_list_max_query_cache_pages_default = 1000;
+static unsigned int prm_list_max_query_cache_pages_flag = 1;
```

Comment:
why `prm_list_query_cache_mode_flag`, `prm_list_max_query_cache_entries_flag`, `prm_list_max_query_cache_pages_flag ` are set 1?

Reply:
Ok, I understood the flag's usage.